src/Microsoft.HttpRepl.Tests/Commands/SetSwaggerCommandTests.cs

```diff
@@ -232,6 +233,40 @@ public async Task ExecuteAsync_WhenThirdSectionIsNotAValidUri_WritesToConsoleMan
             VerifyErrorMessageWasWrittenToConsoleManagerError(shellState);
         }
 
+        [Fact]
+        public async Task ExecuteAsync_WithBaseAddress_DoesNotSetHttpStateBaseAddress()
+        {
+            string response = @"{
+  ""swagger"": ""2.0"",
+  ""host"": ""localhost"",
+  ""schemes"": [
+    ""https""
+  ],
+  ""basePath"": ""/api/v2"",
+  ""paths"": {
+    ""/pets"": {
+    }
+  }
+}";
+            ArrangeInputs(commandText: "set swagger http://localhost/swagger.json",
+                baseAddress: "http://localhost/",
+                path: "/",
+                urlsWithResponse: new Dictionary<string, string> { { "http://localhost/swagger.json", response } },
+                out MockedShellState shellState,
+                out HttpState httpState,
+                out ICoreParseResult parseResult,
+                out _,
+                out _);
+
+            SetSwaggerCommand setSwaggerCommand = new SetSwaggerCommand();
+
+            Assert.Equal("http://localhost/", httpState.BaseAddress.ToString(), StringComparer.Ordinal);
```

Comment:
is this worth asserting over?  Having a pre-assert kind of throws off AAA readability.

Reply:
Removed.